### DISCUSSION TITLE
Misunderstanding

### DISCUSSION THREAD

**Discussion by pithecantrope** [codewars/codewars.com#2806] [Q&A]
Hello, dear mods! Yesterday, after a lot of work, I completed 2 kyu kata (https://www.codewars.com/kata/582c1092306063791c000c00). And I got something like 2 honor points from this. This hit my motivation very hard, because I expected a good reward for good work, but this did not happen. Explain to me: is this a bug, or have I misunderstood / done something? Thanks a lot.

**Comment by hobovsky** [comment]
What is your Codewars user name, and what language did you complete the kata?

**Reply by pithecantrope** [reply]
FartDraft, Python.

**Reply by hobovsky** [reply]
I just checked and all your points earned by solving this kata are there.
Are you sure that some points are still missing since yesterday? It does happen sometimes that points are not visible right after solving a kata (even though they are awarded correctly, they just do not show up). It's a caching issue, its main symptom is that you get only 2 points no matter the rank of the solved kata, and usually lasts only a couple of hours. If you noticed it yesterday, the points should have appeared until today. Looking at your stats, I see that everything is OK.
Are you sure that your points are still missing since yesterday?

**Reply by pithecantrope** [reply]
Thanks, I guess I screwed up myself. Sorry about that. Have a good day!
